Client/WarFare/UIVarious.cpp

```diff
@@ -529,6 +529,7 @@ CUIKnights::CUIKnights()
 	m_pBtn_Appoint = NULL;
 	m_pBtn_Remove = NULL;
 	m_pBtn_Refresh = NULL;
+	m_pBtn_ClanParty = NULL;
```

Comment:
Use nullptr instead of NULL; existing cases need to be updated, so we don't want to add to them.